tests/integration/test_babysit.py

```diff
@@ -295,14 +295,18 @@ def test_check_pending_testing_farm_runs(created):
     check_pending_testing_farm_runs()
 
 
-def test_check_pending_testing_farm_runs_timeout():
+@pytest.mark.parametrize(
+    "status",
+    [TestingFarmResult.new, TestingFarmResult.queued, TestingFarmResult.running],
+)
+def test_check_pending_testing_farm_runs_timeout(status):
     run = flexmock(
         pipeline_id=1,
         status=TestingFarmResult.running,
```

Comment:
I may be missing something but I don't see the `status` argument used anywhere in the test. Should this rather be `status=status`?

Reply:
Indeed. Forgot to update that line. Thanks for noticing!